Current mingw32 doesn't provide <sysinfoapi.h>, so use older API.
--- jasper-4.2.9/src/libjasper/base/jas_malloc.c.orig	2026-03-05 16:39:38.000000000 +0100
+++ jasper-4.2.9/src/libjasper/base/jas_malloc.c	2026-04-28 17:43:45.439110242 +0200
@@ -94,7 +94,6 @@
 #	include <unistd.h>
 #elif defined(_WIN32)
 #	include <windows.h>
-#	include <sysinfoapi.h>
 #endif
 
 /******************************************************************************\
@@ -657,16 +656,13 @@ size_t jas_get_total_mem_size()
 	long page_size = sysconf(_SC_PAGE_SIZE);
 	return pages * page_size;
 #elif defined(_WIN32)
-	/*
-	Reference:
-	https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getphysicallyinstalledsystemmemory
-	*/
-	ULONGLONG mem_size_in_kb;
-	if (!GetPhysicallyInstalledSystemMemory(&mem_size_in_kb)) {
+	MEMORYSTATUSEX memStatusEx = {0};
+	memStatusEx.dwLength = sizeof(MEMORYSTATUSEX);
+	BOOL bRet = GlobalMemoryStatusEx(&memStatusEx);
+	if (!bRet) {
 		return 0;
 	}
-	return (mem_size_in_kb < SIZE_MAX / JAS_CAST(size_t, 1024)) ?
-	  JAS_CAST(size_t, 1024) * mem_size_in_kb : SIZE_MAX;
+	return memStatusEx.ullTotalPhys;
 #else
 	return 0;
 #endif
